.github/actions/test-component/action.yml

```diff
@@ -1,5 +1,5 @@
-name: test specific sktime component
-description: run unit tests on individual component of sktime framework
+name: ${{ inputs.sktime-component-identifier }}, ${{ inputs.python-version-identifier }}
```

Comment:
Does it work? Just confirming, as I'm not sure if inputs are available before name is being defined.

Reply:
I think it does not

Reply:
It did not, so I reverted